cpu/atmega_common/periph/timer.c

```diff
@@ -58,6 +62,7 @@ typedef struct {
     void *arg;                  /**< interrupt callback argument */
     uint8_t mode;               /**< remember the configured mode */
     uint8_t isrs;               /**< remember the interrupt state */
+    int8_t num;                /** < hardware timer number */
```

Comment:
This is an approach where I map the timer within RIOT to the actual hardware timer. Any better solutions?